lib/utils/display.js

```diff
@@ -1,174 +1,207 @@
 const { inspect } = require('util')
-const npmlog = require('npmlog')
-const log = require('./log-shim.js')
+const proggy = require('proggy')
+const log = require('proc-log')
 const { explain } = require('./explain-eresolve.js')
+const { formatWithOptions, format } = require('./format')
+
+const COLOR_PALETTE = ({ chalk: c }) => ({
+  heading: c.white.bgBlack,
```

Comment:
setting the background is a big no-no here. https://github.com/npm/npmlog/pull/56

Reply:
I'm just trying to replicate what npmlog is doing currently. I think changing colors should be a later PR. It looks to me like npmlog is already setting headings with a bg color: https://github.com/npm/npmlog/blob/5e12fe765992964d44b24974925f611aa056b60a/lib/log.js#L384

Reply:
As long as we're not introducing NEW background colors, ok.  We will definitely be doing a color output assessment as the very next step.